ui/src/app/components/multi-toast-banner.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { useState } from 'react';
+
+import colors from 'app/styles/colors';
+
+import { FlexColumn, FlexRow } from './flex';
+import { ArrowLeft, ArrowRight } from './icons';
+import { MultiToastMessage } from './multi-toast-message.model';
+import { ToastBanner } from './toast-banner';
+
+interface Props {
+  messages: MultiToastMessage[];
+  baseZIndex?: number;
+  onDismiss: (messageId: string) => void;
+}
+
+export const MultiToastBanner = ({
+  messages,
+  baseZIndex = 1000,
+  onDismiss,
+}: Props) => {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  if (messages.length === 0) {
+    return null;
+  }
+
+  const currentMessage = messages[currentIndex];
+  const hasMultipleMessages = messages.length > 1;
+
+  return (
+    <FlexColumn>
+      <ToastBanner
+        key={currentMessage.id}
+        title={currentMessage.title}
+        message={
+          <FlexColumn>
+            <div>{currentMessage.message}</div>
+            {hasMultipleMessages && (
+              <FlexRow
```

Comment:
would it make sense to extract a small pagination component or function here?  To me, that would help make the layout obvious at a glance:
```
<FlexColumn>
  message
  paginationArrows
</FlexColumn>
```